src/tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from polydatum import Service
+from polydatum.dal import PathSegment, DataManager
+
+
+@pytest.fixture
+def path_arg():
+    """
+    A factory fixture for a basic PathSegment for use in testing.
+
+    Normally instantiating this for the first time in code implementation
+    will be done by the DAL itself. Because it's somewhat cumbersome, this
+    fixture helps prevent typos and such.
+    """
+    def _factory(name='example'):
+        return (PathSegment(name=name),)
+    return _factory
+
+
+class UserService(Service):
+    def user_test_method(self):
```

Comment:
remove the word "test" from examples. pytest complains when it finds things that look like tests but actually aren't. You can use "example" or "sample" instead.

Reply:
Don't those collection scanners rely on members starting with `test_`? Easy enough to avoid here, I was just curious.

Reply:
I think so. I've just been baking in the pattern to not use it so I can avoid those errors. It can also confuse humans.